### DISCUSSION TITLE
Victron MPPT support ?

### DISCUSSION THREAD

**Discussion by t0ny-Oh** [fl4p/batmon-ha#45] [Ideas]
Hello,
Many thanks fl4p for your great work !

Do you think it will be possible to easily integrate Victron MPPT support?

**Comment by fl4p** [comment]
It should work, but needs some reverse engineering because Victron closed the MPPT bluetooth API for public access.
I tried, but could not get this to work. When I tested it (about a year ago), Victron has published docs for SmartShunt bluetooth protocol only.

**Comment by 3cees** [comment]
Hi. Are there any news on that topic? I would also be intereted in such an integration.
